3_step4_preparing-input-files/index.md

```diff
@@ -21,30 +21,28 @@ All maps that are needed to run the model are listed in the table of Annex 12.
+LISFLOOD needs to know the size properties of each grid cell (length, area) in order to calculate water *volumes* from meteorological forcing variables that are all defined as water *depths*. By default, LISFLOOD obtains this information from the location attributes of the input maps. This will only work if all maps are in an "equal area" (equiareal) projection, and the map co-ordinates (and cell size) are defined in meters. For datasets that use, for example, a latitude-longitude system, neither of these conditions is met. In such cases you can still run LISFLOOD if you provide two additional maps that contain the length and area of each grid cell
 
 ##### Table x.x Optional maps that define grid size
 
-| **Map**         | **Default name** | **Description**                                              |
-| --------------- | ---------------- | ------------------------------------------------------------ |
-| PixelLengthUser | pixleng.map      | Map with pixel length<br><br> Unit: $[m]$,<br> *Range *of values: map \> 0* |
-| PixelAreaUser   | pixarea.map      | Map with pixel area<br><br>*Unit:* $[m^2]$,<br> *Range of values: map \> 0* |
+| **Map**         |  **Default name**  | **Description**                                              |
+| --------------- | ------------------ | ------------------------------------------------------------ |
+| PixelLengthUser | pixleng.map/nc     | Map with pixel length<br><br> Unit: $[m]$,<br> *Range *of values: map \> 0* |
+| PixelAreaUser   | pixarea.map/nc     | Map with pixel area<br><br>*Unit:* $[m^2]$,<br> *Range of values: map \> 0* |
 
 Both maps should be stored in the same directory where all other input maps are. The values on both maps may vary in space. A limitation is that a pixel is always represented as a square, so length and width are considered equal (no rectangles). In order to tell LISFLOOD to ignore the default location attributes and use the maps instead, you need to activate the special option "*gridSizeUserDefined*", which involves adding the following line to the LISFLOOD settings file:
 
 ```xml
 <setoption choice="1" name="gridSizeUserDefined" \>
 ```
 
-LISFLOOD settings files and the use of options are explained in detail in **Chapter XXXX** of this document.
+LISFLOOD settings files and the use of options are explained in detail in a [dedicated chapter](https://ec-jrc.github.io/lisflood-code/3_step3_preparing-setting-file/) and [aanex](https://ec-jrc.github.io/lisflood-code/4_annex_settings_and_options/) of this document.
 
-#### Naming of meteorological variable maps
+**#### Naming of meteorological variable maps .... SHOULD WE REFER TO THE NETCDF CHAPTER? IS THE NETCDF CHAPTER A COMPLEMENT OR A REPLACEMENT TO THIS SECTION??**
```

Comment:
@StefaniaGrimaldi I'd avoid to put direct questions like this into docs. Remove the comment and then let's discuss between us. If further changes are required we can do it later.

Reply:
OK! Thanks!